### PR TITLE
Coverage: reduce arbitrary differences between submit and submit_local (drop pre-1.0 Compat)

### PR DESCRIPTION
**Pull request by vtjnash**
and simplify the code a bit for Julia v1.0

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,5 @@
 language: julia
 julia:
-  - 0.6
```
Comment (ararslan) [inline .travis.yml]: The package still supports 0.6

Comment (vtjnash) [reply .travis.yml]: not anymore, I used 1.0-only features :P

but seriously, do future versions need to still support 0.6?

Comment (ararslan) [reply .travis.yml]: No, but the REQUIRE file should be updated to reflect that

Comment (vtjnash) [reply .travis.yml]: Ah, ok, I forgot about that one. I think I got a few more references too.

**Pull request merged**